### DISCUSSION TITLE
Will the Build workflow in the MKDocs Material Insiders project be fixed soon?

### DISCUSSION THREAD

**Discussion by purpleclay** [squidfunk/mkdocs-material#6140] [Q&A] [answered]
I have just pulled the latest fork of the insiders project but unfortunately the `build.yml` workflow doesn't work. The YAML contains a git merge conflict within it, see: https://github.com/squidfunk/mkdocs-material-insiders/blob/master/.github/workflows/build.yml#L94

**Comment by squidfunk** [comment]
That's unintentional. Thanks for pointing it out! I'll take care of it immediately.

**Reply by squidfunk** [reply, marked as answer]
Fixed, please try again, and sorry for the inconvenience this was causing!
